coverage/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/coverage/artifacts"
+	"os"
+)
+
+const (
+	keyCovProfileFileName    = "key-cov-prof.txt"
+	defaultStdoutRedirect    = "stdout.txt"
+	defaultCoverageTargetDir = "./pkg/"
+	defaultGcsBucket         = "knative-prow"
+	defaultPostSubmitJobName = "post-knative-serving-go-coverage"
+	defaultCovThreshold      = 80
+)
+
+var (
```

Comment:
Why are these `var` and not `const`?

Reply:
it's clear in the code that these values are overwritten if corresponding flag values exist